src/components/dashboard/Overview/Overview.tsx

```diff
@@ -100,6 +100,8 @@ const Overview = (): ReactElement => {
   // Native token is always returned even when its balance is 0
   const tokenCount = useMemo(() => balances.items.filter((token) => token.balance !== '0').length, [balances])
   const nftsCount = useMemo(() => (nfts ? `${nfts.next ? '>' : ''}${nfts.results.length}` : ''), [nfts])
+  // Only initially the items are completely empty because our gateway always returns the native token balance
+  const isTokenCountLoading = balances.items.length === 0
```

Comment:
If this is enough, maybe just keep this? The isNative check is not really needed then.